packages/webui/src/testing/mockTraces/gql.ts

```diff
@@ -4,7 +4,7 @@ import { elapseTime, requestData } from './util';
 
 // GQL query (people)
 const gqlQuery: Event = {
-  id: '2',
+  id: 'TBC',
```

Comment:
💬 IDs in trace data now become irrelevant, because we rewrite the IDs to be sequential in the `mockTraces/index.ts` file.